### ISSUE TITLE
Cursor reaction is not consistent across all clickable elements

### ISSUE THREAD

**Issue by rfirebaugh** [mozilla-iam/dino-park-front-end#50]
Currently when you mouse over + or - or (contact me) buttons the cursor does not change visual state. Yet when you mouse over a name or title or View Full Profile button the cursor does change into a finger. Can this be consistent for all desktop interaction that are clickable? See attached zip file with video in it. 

[Archive.zip](https://github.com/mozilla-iam/dino-park-front-end/files/2475067/Archive.zip)

**Comment by hidde** [comment]
This is because the collapsing/expanding action is a button (does something) and the View Full Profile is a link (goes somewhere).

Cursors match browser and OS defaults (try buttons and links in other parts of Mac OS and you'll see they are the same there). 

I'd recommend keeping cursors as they are, as the link cursor shows it is a link (and users can know, intuitively or otherwise, they can do link-like things with them, like open in new window), not a button, and vice versa.

See also [Some pointers on default cursors](https://hiddedevries.nl/en/blog/2016-08-06-some-pointers-on-default-cursors)

**Comment by rfirebaugh** [comment]
@hidde Yes all clickable links and buttons should trigger a finger cursor.

**Comment by hidde** [comment]
OK. Again, I really think we should not change what cursors look like by default, but if we are sure it is what we want to do, I'll update the CSS.

**Comment by rfirebaugh** [comment]
@hidde @mbransn I did read the article you linked. The article's examples it gives of buttons where the cursor does not change is the operating system interface and browser interface. It goes onto stating what css and apple standards recommend. Take a look at any website with buttons and links even the website which posted this article its buttons the cursor changes to a finger.
